sale_medical_prescription/tests/test_medical_pharmacy.py

```diff
@@ -16,6 +16,7 @@ def setUp(self):
 
     def test_compute_verified_by_id(self):
         """ Test verified_by_id properly set to user """
+        self.pharmacy_4.is_verified = True
```

Comment:
Hmm. Well these tests were failing in 10.0, and these changes fixed them, so I thought that there may be a possibility they fail at some point in 9.0.